src/index.html

```diff
@@ -8,6 +8,107 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
```

Comment:
```suggestion
      <div class="stars__star"></div>
      
      <div class="stars__star"></div>
      
      <div class="stars__star"></div>
      
      <div class="stars__star"></div>
      
      <div class="stars__star"></div>
```
no need to make empty lines inside div
it can be empty lines between siblings